infra/dockerfile.commit_retrieval

```diff
@@ -2,18 +2,21 @@ FROM mozilla/bugbug-base:latest
 
 COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
-# Mercurial need Python2 :(
```

Comment:
This comment still applies, can you bring it back? (and move it to before `# git and git-hyper-blame ...`)

Reply:
Ok, Will do that in the next edit